src/code-templates/package.json

```diff
@@ -8,6 +8,7 @@
     "build": "tsc",
     "build:watch": "tsc --watch",
     "lerna": "lerna",
+    "lint": "eslint '**/*.{js,jsx,ts}'",
```

Comment:
I think we don't need jsx here

Reply:
@rluvaton @DanielGluskin Should not we specify the includes/excludes in eslintrc (where all the definition is located) and here just specify "."?